data.py-dist

```diff
@@ -11,7 +11,7 @@ HOST_DEFAULT_PASSWORD = ""
 # The following prefix will be added to the `name-label` parameter of XAPI objects
 # that the tests will create or import, such as VMs and SRs.
 # Default value: [your login/user]
-# OBJECTS_NAME_PREFIX = "[TEST]"
+OBJECTS_NAME_PREFIX = None
```

Comment:
It seems a good idea to still keep the example as a comment

Reply:
done